src/test/resources/test_cybs.properties

```diff
@@ -12,7 +12,7 @@ timeout=1000
 # logging should normally be disabled in production as it would slow down the
 # processing.  Enable it only when troubleshooting an issue.
 enableLog=true
-logDirectory=<-- Directory where cybs.log file needs to be created -->
+logDirectory=/Users/anetdeveloper/logs
 logMaximumSize=10
 
 
```

Comment:
As part of our changes,we have not touched this file.
Anyway removed that data from the properties.